microcosm_pubsub/caching/resource_cache.py

```diff
@@ -0,0 +1,27 @@
+from microcosm.api import defaults, typed
+from microcosm.config.types import boolean
+from microcosm_pubsub.caching.memcached import MemcachedCache
+
+
```

Comment:
This is the actual microcosm entry point (defined in setup.py see below) that we can hook into with config values e.g. from ansible on a per-service level